pisek/__main__.py

```diff
@@ -74,6 +74,15 @@ def main(argv):
         )
     )
 
+    def add_argument_verbosity(parser):
+        parser.add_argument(
+            "--verbosity",
+            "-v",
+            type=int,
+            default=0,
+            help="Be more verbose.",  # TODO
+        )
```

Comment:
Idea: In most commands, increasing verbosity further is done by using `-v` multiple times. Do we want that? `argparse` supports it with `action='count'`.